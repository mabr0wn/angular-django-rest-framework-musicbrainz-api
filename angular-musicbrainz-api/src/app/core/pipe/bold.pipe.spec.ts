import {BoldPipe} from './bold.pipe';

describe('Pipe: Bold', () => {
    let pipe: BoldPipe;
    let search: string;

    beforeEach(() => {
        pipe = new BoldPipe();
    });

    it('should be instanciated', () => {
        expect(BoldPipe).toMatchSnapshot();
    });
    // text from sample-results.ts
    it('providing text search returns bold pattern', () => {
        expect(pipe.transform('what', 'time is it'))
            .toMatchSnapshot('<b>what</b>', 'time is it');
    });
    // random text...
    it('should allow searching against any test', () => {
        expect(pipe.transform('fee', 'fi fo fum')).toMatchSnapshot('<b>fee</b>', 'fi fo fum');
    });
});
