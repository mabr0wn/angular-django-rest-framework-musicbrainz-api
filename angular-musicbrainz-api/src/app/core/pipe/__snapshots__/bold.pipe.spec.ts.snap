// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pipe: Bold providing text search returns bold pattern: <b>what</b> 1`] = `"<b>what</b>"`;

exports[`Pipe: Bold should allow searching against any test: <b>fee</b> 1`] = `"<b>fee</b>"`;

exports[`Pipe: Bold should be instanciated 1`] = `[Function]`;
